code/src/main/java/com/onlinecontacttracing/messaging/EmailSubject.java

```diff
@@ -0,0 +1,30 @@
+package com.onlinecontacttracing.messaging;
+
+/*
+* System's opening message.
```

Comment:
Plz give a better javadoc comment?

Also I think this class should implement two interfaces :) and that means the methods should have overrides. I also think it should probably be an Enum (similar to the way we did other message types.